psycop/common/feature_generation/data_checks/flattened/test_feature_describer.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import polars as pl
+import pytest
+from polars.testing import assert_frame_equal
+
+from psycop.common.feature_generation.data_checks.flattened.feature_describer_tsflattener_v2 import (
+    ParsedPredictorColumn,
+    generate_feature_description_df,
+    parse_predictor_column_name,
+    tsflattener_v2_column_is_static,
+)
+
+
+@pytest.mark.parametrize(
+    "predictor_column_example",
+    [
+        ParsedPredictorColumn(
+            col_name="pred_value_1_within_0_to_730_days_mean_fallback_nan",
+            feature_name="value_1",
+            fallback="nan",
+            time_interval_start="0",
+            time_interval_end="730",
+            time_interval_format="days",
+            resolve_multiple_strategy="mean",
+            is_static=False,
+        ),
+        ParsedPredictorColumn(
+            col_name="pred_female_fallback_nan",
+            feature_name="female",
+            fallback="nan",
+            time_interval_start="N/A",
+            time_interval_end="N/A",
+            time_interval_format="N/A",
+            resolve_multiple_strategy="N/A",
+            is_static=True,
+        ),
+    ],
+)
+def test_parse_predictor_column_name(predictor_column_example: ParsedPredictorColumn):
+    parsed_col = parse_predictor_column_name(predictor_column_example.col_name)
+    assert parsed_col == predictor_column_example
+
+
+def test_tsflattener_v2_is_static():
+    assert (
+        tsflattener_v2_column_is_static("pred_value_1_within_0_to_730_days_mean_fallback_nan")
+        is False
+    )
+    assert tsflattener_v2_column_is_static("pred_female_fallback_nan") is True
+
+
+def test_generate_feature_description_df():
+    df = pl.DataFrame(
+        {
+            "pred_female_fallback_nan": [0, 1, np.nan],
+            "outc_first_scz_or_bp_within_0_to_730_days_max_fallback_0": [0, 0, 1],
+            "pred_value_1_within_0_to_730_days_mean_fallback_nan": [1, 2, 3],
```

Comment:
Nitpick: Would parametrise these, testing one column at a time. Makes the "expected" df easier to read.